code/game/objects/items/cosmetics.dm

```diff
@@ -243,6 +243,7 @@
 					to_chat(user, span_warning("[human_target] is just way too shaved. Like, really really shaved."))
 					return
 				user.visible_message(span_notice("[user] tries to change [human_target]'s facial hairstyle using [src]."), span_notice("You try to change [human_target]'s facial hairstyle using [src]."))
+				playsound(src, 'sound/items/hair-clippers.ogg', 50)
```

Comment:
there's nothing stopping the 5 sec audio clip playing if the do_after is interrupted so it could get spammed hard

Reply:
This item is relatively unused and rare. Plenty of sounds (hundreds) are linked to items that fall into the same category of spammable. The only thing that has a sound.stop() sound.start() are machinery looping sounds.

Reply:
if it were me, I'd shorten the sound to 1-2 seconds and only play it if the do_after was successful